package.json

```diff
@@ -2,8 +2,26 @@
   "name": "angle-multisig",
   "version": "1.0.0",
   "description": "",
+  "scripts": {
+    "address": "rm -rf ./build && tsc && node ./build/scripts/hardhat/getAddress.js",
+    "execute": "rm -rf ./build && tsc && node ./build/scripts/main.js",
+    "deploy:fork": "forge script --fork-url fork --broadcast",
+    "fork": "source .env && anvil --fork-url $ETH_NODE_URI_MAINNET",
+    "getSignatures": "rm -rf ./build && tsc && node ./build/scripts/getSignatures.js",
+    "merkle": "rm -rf ./build && tsc && node ./build/scripts/updateAirdropMerkle.js",
+    "path": "rm -rf ./build && tsc && node ./build/scripts/encodePath.js",
+    "removeStaking": "rm -rf ./build && tsc && node ./build/scripts/removeStaking.js",
```

Comment:
these look like core module related? Should we remove it?

Reply:
The fork commands? No it is to verify the transaction will pass
The flow is:
- fork the network where you want to push the tx
- create the .sol scripts 
- test the tx and check it has the expected behaviour
- create the .json 
- execute `submitFoundryTx.ts`

Reply:
No the ones linked to hardhat files -> like getSignatures, removeStaking...

Reply:
Ah yes I will delete when I delete the .ts files